packages/ZbayLite/src/rtl-tests/community.join.test.tsx

```diff
@@ -51,6 +52,14 @@ describe('User', () => {
 
     const factory = await getFactory(store)
 
+    const quietCommunity = await factory.create<
```

Comment:
The very important thing about using factory-girl is that it saves value to the store. There are two methods: `build` and `create`. The first one builds the model but it **doesn't save** it to the store - and that's the desired behavior in this case (we test real-life-like scenario in which we want to modify the store only with the data that comes from the mocked waggle). At this moment, it turns out that we have two communities in this test